metadata/reportssql/report_util_functions.sql

```diff
@@ -320,21 +296,7 @@ CREATE FUNCTION patientHasStartedARVTreatmentBeforeExtendedEndDate(
     p_extendedMonths INT(11)) RETURNS TINYINT(1)
     DETERMINISTIC
 BEGIN
-    DECLARE result TINYINT(1) DEFAULT 0;
-    DECLARE uuidARVTreatmentStartDate VARCHAR(38) DEFAULT "e3f9c7ee-aa3e-4224-9d18-42e09b095ac6";
-
-    SELECT
-        TRUE INTO result
-    FROM obs o
-    JOIN concept c ON c.concept_id = o.concept_id AND c.retired = 0
-    WHERE o.voided = 0
-        AND o.person_id = p_patientId
-        AND c.uuid = uuidARVTreatmentStartDate
-        AND o.value_datetime IS NOT NULL
-        AND timestampadd(MONTH, -p_extendedMonths, cast(o.value_datetime AS DATE)) < p_endDate
-    GROUP BY c.uuid;
-
-    RETURN (result);
+    RETURN timestampadd(MONTH, -p_extendedMonths, getPatientDateOfEnrolmentInHIVProgram(p_patientId)) < p_endDate;
```

Comment:
@tmvumbi2 This function used to return 0 but now it returns NULL when the date is null.